index.js

```diff
@@ -349,6 +350,10 @@ class Squeakquel extends Datastore {
             });
         }
 
+        if (config.sortBy) {
+            sortKey = config.sortBy;
```

Comment:
what if `sortBy` is an invalid key? Do we need to check?

Reply:
I can work on a fix in the data-schema
Meanwhile, I think we should merge these other PRs to test if this call even works first in beta